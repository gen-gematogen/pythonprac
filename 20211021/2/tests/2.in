:foo y print(y)
foo 'Hello'
quit "How're_u_doin_dude_â„–_{}.{}?"