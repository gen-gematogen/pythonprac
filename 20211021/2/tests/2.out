Hello
None
How're_u_doin_dude_â„–_2.3?
